.evg.yml

```diff
@@ -320,6 +322,104 @@ tasks:
           content_type: application/octet-stream
           permissions: public-read
 
+  - name: release_tag
+    depends_on:
+      - name: lint
+      - name: test_unit
+      - name: test_with_cloud
+      - name: test_e2e
+      - name: build_publish_clis
+    patchable: false
+    commands:
+      - func: "setup_node"
+      - func: "setup_project"
+      - command: s3.get
+        params:
+          aws_key: ${artifacts_aws_key}
+          aws_secret: ${artifacts_aws_secret}
+          remote_file: versions/cloud-prod/CURRENT
+          bucket: realm-clis
+          local_file: manifest.json
+      - command: shell.exec
+        params:
+          shell: "bash"
+          script: |
+            set -e
+            cd realm-cli
+            if [ "${branch_name}" != "master" ]; then
+              echo "skipping release: not on master branch"
+              exit 0
+            fi
+            if [ $(git tag -l "v${cli_version}") ]; then
+              echo "skipping release: tag already exists"
+              exit 0
+            fi
+            touch release.true
+      - command: shell.exec
+        params:
+          shell: "bash"
+          script: |
+            set -e
+            if [ ! -f realm-cli/release.true ]; then
+              # not a release build
+              exit 0
+            fi
```

Comment:
we should be able to simplify this and remove it completely. 
since we don't need to add the npm creds _yet_, we can remove L357-366